src/test/java/com/amazon/opendistroforelasticsearch/security/dlic/rest/api/UserApiTest.java

```diff
@@ -470,12 +467,12 @@ public void testUserApiWithDots() throws Exception {
         rh.keystore = "restapi/kirk-keystore.jks";
         rh.sendAdminCertificate = true;
 
-        // initial configuration, 5 users
+        // initial configuration, 6 users
         HttpResponse response = rh
                 .executeGetRequest("_opendistro/_security/api/" + CType.INTERNALUSERS.toLCString());
         Assert.assertEquals(HttpStatus.SC_OK, response.getStatusCode());
         Settings settings = Settings.builder().loadFromSource(response.getBody(), XContentType.JSON).build();
-        Assert.assertEquals(49, settings.size());
+        Assert.assertEquals(56, settings.size());
```

Comment:
Any reason why these values changed?

Reply:
I added another user to test whitelisting, so the value changed